code-challenges.test.js

```diff
@@ -21,8 +31,29 @@ const secretCodeWord2 = "Gobbledygook"
 const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
+describe("decoder", () => {
+    it ("takes in a string and returns a string with a coded message.", () => {
+      expect(decoder(secretCodeWord1)).toEqual(["L4ck4d41s1c4l"]) 
+      expect(decoder(secretCodeWord2)).toEqual(["G0bbl3dyg00k"])
+      expect(decoder(secretCodeWord3)).toEqual(["3cc3ntr1c"])
+    })
+  })
+
+//   FAIL  ./code-challenges.test.js
+//   decoder
+//     ✕ takes in a string and returns a string with a coded message. (1 ms)
+
+//   ● decoder › takes in a string and returns a string with a coded message.
+
+//     ReferenceError: decoder is not defined
+
 // b) Create the function that makes the test pass.
 
+const decoder = (word) => {
+let newWord = word.replace(/a/, "4")
+// this is right where i got stuck i wasn't sure how to implement this method or if i was even supposed to use this one in particular
```

Comment:
So for this I want you to take a look at global (g) case insensitive (i
). I will even attach the first documentation that you can check out. You are really close here and I want you to take another look at it.